gulp/package-task.js

```diff
@@ -20,6 +20,11 @@ export const packageTask = ((cache) => memoizeTask(cache, function bundle(target
 
 export default packageTask;
 
+const sideEffects = [
+  "./asynciterable/todomstream.mjs",
+  "./asynciterable/todomstream.js"
+];
+
```

Comment:
Agree with this change, the prototype patching needs to be captured. We should probably make this recursively read and add all the files in [`src/add`](https://github.com/ReactiveX/IxJS/tree/master/src/add), since prototype patching is their entire purpose.

Reply:
I have adjusted the implementation to also dynamically add all files from `src/add` [here](https://github.com/ReactiveX/IxJS/pull/383/files#diff-8e11d351013aa05c89d7b93a85925d579c6242bf672f3f558df515277348d735R25-R44).

I just used `glob` for this. Please let me know if that is good enough from the perspective of integrating with gulp.